modules/svneverever/__main__.py

```diff
@@ -255,12 +256,17 @@ def command_line():
 
     return args
 
+def login(realm, username, may_save):
+    username = six.input("Please enter your username: ")
+    password = getpass.getpass()
```

Comment:
The default prompt of `getpass` is `Password: `. Let's make these two match more regarding output style, either
```
username = six.input('Username: ')
password = getpass.getpass()
```
or even
```
username = six.input('Username: ')
password = getpass.getpass('Password: ')
```